### ISSUE TITLE
on change event 

### ISSUE THREAD

**Issue by dedihartono** [TotomInc/vue3-select-component#141]
Hi, looks like on change event doesn't exist, 
event thougt we can use v-model

**Comment by TotomInc** [comment]
You're right, I'm not a fan of adding these kind of events as most of the time, they are used incorrectly and cause side-effects.

Also, this can be easily achieved using a watch on the v-model value, but I agree this isn't convenient.

What would be the use case of this `@change` event for you?

**Comment by dedihartono** [comment]
oh okay, I found it, i could use @option-selected on my case,
thank you
